cmd/agent/app/agent.go

```diff
@@ -309,9 +313,11 @@ func registerWithControlPlaneAPIServer(controlPlaneRestConfig, clusterRestConfig
 	}
 
 	impersonatorSecret.OwnerReferences = []metav1.OwnerReference{
-		*metav1.NewControllerRef(clusterObj, clusterv1alpha1.SchemeGroupVersion.WithKind("Cluster"))}
+		*metav1.NewControllerRef(clusterObj, clusterv1alpha1.SchemeGroupVersion.WithKind("Cluster")),
+	}
 	impersonatorSecret.Data = map[string][]byte{
-		clusterv1alpha1.SecretTokenKey: clusterImpersonatorSecret.Data[clusterv1alpha1.SecretTokenKey]}
+		clusterv1alpha1.SecretTokenKey: clusterImpersonatorSecret.Data[clusterv1alpha1.SecretTokenKey],
+	}
```

Comment:
What's changed here?

Reply:
It's modified by my vscode extension automatically, I guess it's because I set
`gopls.formatting.gofumpt: true`

Reply:
Can you revoke the change? Because it will pollute the code history and bring distraction from reviewing.

Reply:
OK, reverted